### PR TITLE
Get adapter from the initializer

### PR DESCRIPTION
**Pull request by sunayna**

### CODE REVIEW

File: src/java/com/sarality/app/view/list/ListViewInitializer.java [hunk 1]
```diff
@@ -64,4 +65,8 @@ public void setupActions() {
       return new ListViewAdapter<T>(getContext(), dataList, getRowRenderer());
     }
   }
+
+  public ListViewAdapter<T> getAdapter() {
```
Comment (abhideep) [inline src/java/com/sarality/app/view/list/ListViewInitializer.java]: why do we need to expose this?

Comment (sunayna) [inline src/java/com/sarality/app/view/list/ListViewInitializer.java]: Need to get the filter from the adapter. Filter is exposed but not adapter.

Comment (abhideep) [inline src/java/com/sarality/app/view/list/ListViewInitializer.java]: The original question was "why" :)?

If we pass in the Filter to the initializer, why do we need the adapter. Just trying to see why we need access to Adapter and if there is some way to accomplish the same without exposing the Adapter.

Comment (sunayna) [inline src/java/com/sarality/app/view/list/ListViewInitializer.java]: best way is to put up the code for review :).  The filter needs to get the input text somehow...
put up new review in dothat/flights

**Pull request closed without merging**